ci-templates/stages.yml

```diff
@@ -346,12 +346,13 @@ stages:
         parameters:
           poolSettings: ${{parameters.poolSettings}}
           expectUseVMs: ${{parameters.expectUseVMs}}
-          timeoutInMinutes: 30
+          timeoutInMinutes: 35
```

Comment:
Uh oh. :slightly_smiling_face:

Reply:
Well this was just done to be on the safe side since I added a new test :). 
I actually didn't see any actual problem :P